### PR TITLE
docs: Wifi debugging steps added

### LINKED ISSUE #18: How to connect the real device over Wifi for debugging
I need to connect my Andriod device over Wifi instead of the USB cable.

### PR DESCRIPTION
**Pull request by yashwp**
Closes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,18 @@
...
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+## Connect real device over Wi-Fi
+1. Connect your device with laptop using USB and connect mobile with same WiFi to which your laptop is connect
+2. Open CMD/Terminal
+3. Check if device correctly connected using `adb devices`
+4. Run `adb tcpip 5555`
+5. Open your mobile `Settings` > `About Phone` > `Status` > `IP Address` (IP Address will change everytime you connect)
+6. Run `adb connect <your_mobile_ip>` eg: - `adb connect 192.168.1.6`
```
Comment (daksheshvira) [inline README.md:18]: you have to rub "adb connect 192.168.1.6:5555" to connect to device

Comment (yashwp) [reply README.md:18]: @daksheshvira I never really got a need to use `adb kill-server`,  `adb usb`. Can please explain what these two will actually do?

Though `adb tcpip 5555` is anyway mentioned in step 4.

Comment (daksheshvira) [reply README.md:18]: @yashwp "adb kill-server kills" the adb debugging and "adb usb" starts the adb debugging for usb and "adb tcpip 5555" starts the network debugging.

Comment (yashwp) [reply README.md:18]: @daksheshvira So, can you create a `patch` to this PR?

**Review by daksheshvira — APPROVED** [review]

**Pull request merged**